crates/rolldown_error/src/events/unresolved_import.rs

```diff
@@ -1,20 +1,54 @@
 use crate::types::diagnostic_options::DiagnosticOptions;
+use arcstr::ArcStr;
+use oxc::span::Span;
 
 use super::BuildEvent;
-use std::path::PathBuf;
+
+#[derive(Debug)]
```

Comment:
The name is also not proper, the `id` and `source` came from `importer`, but the span came from `module_request` which is related to importee, if you cannot find a better naming, just flatten them.

Reply:
They are interrelated, an aggregation of `Option`, and flattening can become very complex. Let's come up with a good name. 🤔☝️

Reply:
How about this one below?
```rust
pub struct UnloadableDependencyContext {
  pub source: ArcStr,
  pub importer_id: ArcStr,
  pub importee_span: Span,
}

```